docs/blog/csed-week/csed-week-2019.md

```diff
@@ -0,0 +1,180 @@
+# MakeCode Tutorials and Projects for CS Ed Week
+
+[Computer Science Education week](https://csedweek.org/) is happening again this year and it's coming soon! We are excited about it and want to provide some quick and easy MakeCode activities for students to use. So, we've updated last year's [list](/blog/csed-week/csed-week-2018) of some of our favorite tutorials, guides, and projects that you can do in under an hour. 
+
+We’ve organized this list by product:
+
+## MakeCode Arcade
+
+![Arcade card](/static/blog/csed-week/arcade-badge.png)
+
+Hey, this year MakeCode Arcade is available! Try out some of these game coding ideas!
+
+### 1. Quick start game coding cards
+
+Easy to follow, quick reference cards for game design and coding concepts -
+
+* [Create a Moving Character](https://drive.google.com/open?id=1QTCMMgikY-SVPbKxh0D_U7gPx3M0zXIS)
+* [Build a Tile Map Level](https://drive.google.com/open?id=1k79d_bg44LN7bZvCizwevF1pCcTtRSYO)
+* [Something to Collect](https://drive.google.com/open?id=17iGmphagmBfx1iW_2ldfoNgkns22Qcr2)
+* [Array of Tiles](https://drive.google.com/open?id=1CIOkUNWBbCiH98gG1veefFIgHwGKcB5O)
+* [Run and Jump Platformer - Part 1](https://drive.google.com/open?id=1KseJtE26o8q-OLDe6CpwpHPHtzbIWYeP)
+* [Run and Jump Platformer - Part 2](https://drive.google.com/open?id=1tWRIUsp007nlwckPto9-POg-L2T7kEP8)
+* [Run and Jump Platformer - Part 3](https://drive.google.com/open?id=1w15idO8VD7_5Ji3zVN036zPqGb5i119-)
+
+### 2. Tutorials
+
+* [Eat the Doughnut](https://arcade.makecode.com/#tutorial:tutorials/eat-the-doughnut)
```

Comment:
needs the leading `/`, I think I might remember it becoming required to load properly when user tutorials (`https://arcade.makecode.com/#tutorial:https://...`) started being supported to disambiguate builtin from random md page / share links

also eat the donut is no more, this one would be chase the pizza

```suggestion
* [Eat the Doughnut](https://arcade.makecode.com/#tutorial:/tutorials/chase-the-pizza)
```

Reply:
Ah, yes. That's working. Thanks.